docs/usage/available-methods-examples.md

```diff
@@ -7,25 +7,25 @@ The library supports all the methods listed on Telegram Bot API docs [page](http
 See [sendMessage](https://core.telegram.org/bots/api#sendmessage) docs for a list of supported parameters and other info.
 
 ```php
-$response = $telegram->sendMessage('CHAT_ID', 'Hello World');
+$response = $telegram->sendMessage(['chat_id' => 'CHAT_ID', 'text' => 'Hello World']);
 $messageId = $response->getMessageId();
 ```
 
-## Send a Photo
+## Forward a Message
 
-See [sendPhoto](https://core.telegram.org/bots/api#sendphoto) docs for a list of supported parameters and other info.
+See [forwardMessage](https://core.telegram.org/bots/api#forwardmessage) docs for a list of supported parameters and other info.
 
 ```php
-$response = $telegram->sendPhoto('CHAT_ID', 'path/to/photo.jpg', 'Some caption');
+$response = $telegram->forwardMessage(['chat_id' => 'CHAT_ID', 'from_chat_id' => 'FROM_CHAT_ID', 'message_id' => 'MESSAGE_ID']);
 $messageId = $response->getMessageId();
 ```
 
-## Forward a Message
+## Send a Photo
 
-See [forwardMessage](https://core.telegram.org/bots/api#forwardmessage) docs for a list of supported parameters and other info.
+See [sendPhoto](https://core.telegram.org/bots/api#sendphoto) docs for a list of supported parameters and other info.
 
 ```php
-$response = $telegram->forwardMessage('CHAT_ID', 'FROM_CHAT_ID', 'MESSAGE_ID');
+$response = $telegram->sendPhoto(['chat_id' => 'CHAT_ID', 'photo' => 'path/to/photo.jpg', 'caption' => 'Some caption']);
```

Comment:
Photo now also supports remote URL. So lets add another example to this showing the same.